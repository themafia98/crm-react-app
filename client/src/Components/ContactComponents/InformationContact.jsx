import React,{Fragment} from 'react';
import './contactComponents.scss';
const InformationContact = () => {


    return (
        <Fragment>
            <p className = 'Contact__content__skype'>
            skype:<span className = 'skype'>Test</span>
            </p>
            <p className = 'Contact__content__telegram'>
            telegram:<span className = 'telegram'>Test</span>
            </p>
            <p className = 'Contact__content__tel'>
            +22 (222) 222 22 22
            </p>
            <p className = 'Contact__content__country'>
                Минск, Беларусь.
            </p>
        </Fragment>
    )
};
export default InformationContact;