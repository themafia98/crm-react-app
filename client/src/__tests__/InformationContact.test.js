import React from 'react';
import renderer from 'react-test-renderer';

import InformationContact from '../Components/ContactComponents/InformationContact';

const InformationContactTest = renderer.create(
    <InformationContact />
);

test('ContactComponents information test', () => {

    let treeSnapshotInformationContact = InformationContactTest.toJSON();
    expect(treeSnapshotInformationContact).toMatchSnapshot();
});