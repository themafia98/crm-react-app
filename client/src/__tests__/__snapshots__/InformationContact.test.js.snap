// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactComponents information test 1`] = `
Array [
  <p
    className="Contact__content__skype"
  >
    skype:
    <a
      className="skype"
      href="skype:Test"
    >
      Test
    </a>
  </p>,
  <p
    className="Contact__content__telegram"
  >
    telegram:
    <a
      className="telegram"
      href="https://telegram.me/holykilla"
      rel="noopener noreferrer"
      target="_blank"
    >
      Test
    </a>
  </p>,
  <a
    className="Contact__content__tel"
    href="tel:+222222222222"
  >
    +22 (222) 222 22 22
  </a>,
  <p
    className="Contact__content__country"
  >
    Минск, Беларусь.
  </p>,
]
`;
